v2/fleet-local/it-hybrid/memcache@.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=Proxy { Memcache } @ %i
+After=docker.service  
+Requires=docker.service
+
+
+[Service]
+User=core
+Restart=always
+RestartSec=5
+TimeoutStartSec=0
+
+ExecStartPre=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && docker pull borja/docker-memcached"
+ExecStartPre=-/usr/bin/docker kill memcache
+ExecStartPre=-/usr/bin/docker rm memcache
+
+# NOTE: it's critical to source the etcdctl.sh file so that etcd connects to the correct cluster.
+ExecStart=/usr/bin/sh -c "source /etc/profile.d/etcdctl.sh && \
+  echo $($CMD) | xargs docker run \
+    --name memcache \
```

Comment:
nitpick... `memcache` or `memcached`?   The original container is `memcached` and that's the name of the daemon.  @iderdik @bryanlatten